src/utils/timers.js

```diff
@@ -3,3 +3,15 @@ export const setTimeoutPromise = async ms => {
     setTimeout(resolve, ms)
   })
 }
+
```

Comment:
could also use https://github.com/sindresorhus/p-timeout

Reply:
No need unless we need the extra features.